apps/nrm/index.js

```diff
@@ -291,11 +291,36 @@ module.exports = {
       ],
       next: '/who-exploited-pv'
     },
+
     '/who-exploited-pv': {
       behaviours: [
         saveFormSession
       ],
       fields: ['who-exploited-pv'],
+      next: '/information-exploiters-location'
+    },
+
+    '/information-exploiters-location': {
+      behaviours: saveFormSession,
+      fields: ['exploiters-location'],
+      forks: [{
+        target: '/exploiters-uk',
+        condition: {
+          field: 'exploiters-location',
+          value: 'yes'
+        }
+      }],
+      next: '/types-of-exploitation'
+    },
+
+    '/exploiters-uk': {
+      behaviours: saveFormSession,
+      fields: ['are-exploiters-in-the-uk'],
+      next: '/where-are-exploiters'
+    },
+    '/where-are-exploiters': {
```

Comment:
I'd make the step name the same as the field name (exploiters-current-location-details), it fits the question more